src/io/sb3/fromSb3.ts

```diff
@@ -441,9 +386,9 @@ export async function fromSb3JSON(json: sb3.ProjectJSON, options: { getAsset: ge
           async (spriteData: sb3.Sprite) =>
             new Sprite({
               name: spriteData.name,
-              costumes: await extractCostumes(spriteData, options.getAsset),
+              costumes: await extractCostumes(spriteData, options.getAssetData),
               costumeNumber: spriteData.currentCostume,
-              sounds: await extractSounds(spriteData, options.getAsset),
+              sounds: await extractSounds(spriteData, options.getAssetData),
```

Comment:
Is there any benefit to running `extractSounds` and `extractCostumes` in parallel, a la:
```js
const [costumes, sounds] = await Promise.all([
    extractCostumes(spriteData, options.getAssetData),
    extractSounds(spriteData, options.getAssetData)
]);
```

Reply:
Probably! Updated, thanks. (Commits to be pushed soon™.)